### PR TITLE
Update Contributor Journey page's title

### PR DESCRIPTION
**Pull request by vishalvivekm**
**Notes for Reviewers**

The page https://layer5.io/community/handbook/contributor-journey
has same title as https://layer5.io/community/handbook and it was coming up in seo site-audit.

This PR Updates Contributor Journey page's title.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 3a3a7f09c520d4d2da5baf1742089aad30c19e6c at: https://64de1ad5f3a3b47ff9cb820b--layer5.netlify.app

File: src/pages/community/handbook/contributor-journey.js [hunk 1]
```diff
@@ -13,5 +13,5 @@ const CommunityAbout = () => {
 };
 export default CommunityAbout;
 export const Head = () => {
-  return <SEO title="Community Handbook" description="Instructions to encourage and support first-time open-source contributors." />;
-};
\ No newline at end of file
+  return <SEO title="Contributor-Journey" description="Instructions to encourage and support first-time open-source contributors." />;
```
Comment (leecalcote) [inline src/pages/community/handbook/contributor-journey.js:16]: Why are you hyphenating?

**Review by leecalcote — CHANGES_REQUESTED** [review]
Provide grammatical reference for your logic behind hyphenating.

File: src/pages/community/handbook/contributor-journey.js (hunk 1 above)
Comment (vishalvivekm) [reply src/pages/community/handbook/contributor-journey.js:16]: @leecalcote 
I saw the file name : `community/handbook/contributor-journey.js` contributor-journey.js and that's y put the title the same.
Shall I change it to `Contributor Journey`?

**Comment by leecalcote** [comment]
@vishalvivekm I'm trying to review too quickly, I think. I'll let @thisiskaransgit or @Chadha93 make a call here.

**Comment by l5io** [comment]
🚀 Preview for commit e35f582685e48e20f25b3ed2f3c19f84eb02af5c at: https://64df6ed68b14bd0af65869a7--layer5.netlify.app

**Review by Chadha93 — APPROVED** [review]

**Pull request merged**